src/pages/workspaces/List.js

```diff
@@ -245,9 +247,15 @@ export const WorkspaceList = _.flow(
     return h(Fragment, [
       h(TopBar, { title: 'Workspaces' }, [h(WsSearch, { onChange: v => this.setState({ filter: v }) })]),
       h(PageBox, { style: { position: 'relative' } }, [
-        div({ style: { display: 'flex', alignItems: 'center', justifyContent: 'space-between', marginBottom: '1rem' } }, [
+        div({ style: { display: 'flex', alignItems: 'center', justifyContent: 'flex-end', marginBottom: '1rem' } }, [
           div({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase' } }, ['Workspaces']),
-          div({ style: { marginLeft: 'auto', flex: '0 0 300px' } }, [
+          div({ style: { marginLeft: 'auto' } }, [
+            h(LabeledCheckbox, {
+              checked: includePublic === true,
+              onChange: v => this.setState({ includePublic: v })
+            }, ' Include public workspaces')
```

Comment:
```suggestion
            }, ' Show public workspaces')
```

Reply:
Nice catch on the persisting! I forgot to save it to state history. I like that the behavior matches now